image-rs/src/unpack.rs

```diff
@@ -22,6 +22,60 @@ use std::{
 use tokio::{fs, io::AsyncRead};
 use tokio_tar::ArchiveBuilder;
 
+const WHITEOUT_PREFIX: &str = ".wh.";
+const WHITEOUT_OPAQUE_DIR: &str = ".wh..wh..opq";
+
+#[cfg(not(any(target_os = "linux", target_os = "freebsd", target_os = "macos")))]
+fn convert_whiteout(path: &Path, uid: u32, gid: u32, mode: Option<u32>, destination: &Path) -> Result<bool> {
+    Ok(true)
```

Comment:
Probably we could use a shared `convert_whiteout` funciton. With `cfg-if` to firstly judge the platform and return `Ok(true)` if needed. Thus we do not need two `convert_whiteout` versions.